[MIG] account_ux: Migration to 15.0

jjscarafia opened the pull request.

jjscarafia committed:
[MIG] account_ux/: Migration to 15.0

roboadhoc commented:
[Pull request status dashboard](https://runbot.adhoc.com.ar/ingadhoc/account-financial-tools/pull/314).

jjscarafia committed:
[REM] account_ux: modification of reconciliation method

Por el momento desactivamos esta funcionalidad hasta evaluar si la vamos a mantener o no a partir de v15

jjscarafia committed:
[REF] account_ux: remove comment about group by
Odoo no implemento agrupar sobre no stored, solo sobre m2m.

jjscarafia commented:
@roboadhoc r+

roboadhoc commented:
Because this PR has multiple commits, I need to know how to merge it:

* `merge` to merge directly, using the PR as merge commit message
* `rebase-merge` to rebase and merge, using the PR as merge commit message
* `rebase-ff` to rebase and fast-forward

jjscarafia commented:
@roboadhoc rebase-ff

roboadhoc commented:
Merge method set to rebase and fast-forward
